nodes/test/test_qdrant_mark_deleted.py

```
@@ -0,0 +1,155 @@
+# ---------------------------------------------------------------------------
+# Bootstrap mocks so qdrant.py can be imported without real dependencies
+# ---------------------------------------------------------------------------
+
+NODES_SRC = Path(__file__).parent.parent / 'src' / 'nodes'
+
+
+# --- ai.common.store: provide a real DocumentStoreBase with stub methods ---
+class _FakeDocumentStoreBase:
+    """Minimal stand-in for ai.common.store.DocumentStoreBase."""
+
+    def __init__(self, *a, **kw):
```

Ruff (0.15.6)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)